api/hello.go

```diff
@@ -8,14 +8,12 @@ import (
 	"github.com/ONSdigital/log.go/v2/log"
 )
 
-// TODO: remove hello world handler
 const helloMessage = "Hello, World!"
```

Comment:
Why have the comments and godoc been removed from this file? They are there as a reminder that HelloWorld is not permanent and only for the skeleton service. The godoc is particularly relevant as it describes what the public method is doing.

Reply:
👍